packages/suite/src/views/dashboard/components/T2B1PromoBanner.tsx

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import styled from 'styled-components';
+import { rgba } from 'polished';
+
+import { EventType, analytics } from '@trezor/suite-analytics';
+import { TREZOR_URL } from '@trezor/urls';
+
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { selectT2B1PromoBanner } from 'src/reducers/suite/suiteReducer';
+import { Button, Image, variables } from '@trezor/components';
+import { setFlag } from 'src/actions/suite/suiteActions';
+import { Translation, TrezorLink } from 'src/components/suite';
+
+// banner size 800px
+const SMALL_SCREEN_BREAKPOINT = 832;
+// banner size 640px
+const MOBILE_BREAKPOINT = 671;
+
+const BannerWrapper = styled.div`
+    display: grid;
+    grid-template-columns: 338px 1fr minmax(109px, 145px) 42px;
+    grid-template-rows: 42px 1fr 0;
+    column-gap: 8px;
+    background-color: #0f6148;
```

Comment:
would be great if you cherry pick [this](https://github.com/trezor/trezor-suite/commit/df7773c1fc248c16f750299be827a670450336fa) commit and simply use `backgroundPrimaryDefault` here.